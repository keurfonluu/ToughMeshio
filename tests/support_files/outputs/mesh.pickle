version https://git-lfs.github.com/spec/v1
oid sha256:2c835c257fe6770b4a515abb944d455e8e959af68f2a9b73736f1fbc174824ca
size 145710
