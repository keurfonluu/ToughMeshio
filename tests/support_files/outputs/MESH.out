version https://git-lfs.github.com/spec/v1
oid sha256:f1cbdec2b56f7a2fd56bbdc285c19015c93be9dbc11282efd640a93cc90c5d5b
size 199248
